src/server/typingsInstaller/typingsInstaller.ts

```diff
@@ -286,7 +292,7 @@ namespace ts.server.typingsInstaller {
                     this.log.writeLine(`Npm config file: '${npmConfigPath}' is missing, creating new one...`);
                 }
                 this.ensureDirectoryExists(directory, this.installTypingHost);
-                this.installTypingHost.writeFile(npmConfigPath, "{}");
+                this.installTypingHost.writeFile(npmConfigPath, JSON.stringify({ typeScriptVersion: ts.version }));

```

Comment:
consider extracting the `{ typeScriptVersion: ts.version }` to a function and using it here and in `writePackageVersion`